app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_comment_counter
+
+  def update_comment_counter
+    post.increment!(:comments_counter)
+  end
```

Comment:
- [ ] It is considered good practice to declare callback methods as private. If left public, they can be called from outside of the model and violate the principle of object encapsulation. Kindly, make ```update_comment_counter``` private.